.golangci.yml

```diff
@@ -18,15 +18,6 @@ run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   deadline: 5m
 
-  # which dirs to skip: they won't be analyzed;
-  # can use regexp here: generated.*, regexp is applied on full path;
-  # default value is empty list, but next dirs are always skipped independently
-  # from this option's value:
-  #   	vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  skip-dirs:
-    - pkg/binary/envoytest
```

Comment:
I took out the skip-dirs thing from here as it is handled externally. we aren't vendoring anything and addressing or ignoring the lint problems in envoytest was imho a better call than the confusion of two things doing skip-dir (make and here)